src/network/networkprotocol.cpp

```diff
@@ -71,10 +71,13 @@
 	PROTOCOL VERSION 50
 		Support for TOCLIENT_SPAWN_PARTICLE_BATCH
 		[scheduled bump for 5.14.0]
+	PROTOCOL VERSION 51
+		Only send first frame of animated item/wield images to older client
+		[scheduled bump for 5.15.0]
 */
 
 // Note: Also update core.protocol_versions in builtin when bumping
-const u16 LATEST_PROTOCOL_VERSION = 50;
+const u16 LATEST_PROTOCOL_VERSION = 51;
```

Comment:
Design question to the dev team: Should we bump already, or in the 5.15.0-rc build?

Reply:
I am mainly asking this in case there would be a second PR for 5.15.0 that also increases the version. I guess it does not matter. We could bump it multiple times per version.